doc/nrf/ug_thread.rst

```diff
@@ -379,39 +340,12 @@ Antenna diversity
 The antenna diversity module allows the 802.15.4 radio device to select the appropriate antenna for transmission or reception, when the device is equipped with two antennas.
 This can help improve the radio communication parameters in difficult radio environments.
 
-Radio ANT pin is connected to those two antennas through FEM (Front End Module), which provides single pin to act as a binary switch, so that the antenna may be selected freely.
-
-.. figure:: images/ant_diversity_interface.svg
-   :alt: Multiple antenna setup for antenna diversity module
-
-   Multiple antenna setup for antenna diversity module
-
-There is a number of approaches that can be then used to switch the antenna to minimize the effect of phenomena that negatively impact radio communication.
-In case of reception, the approach implemented by this module is based on RSSI measurements during the preamble of a received frame.
-
-Moreover, an interface is provided for manual antenna selection, if other criteria for selecting the antenna are required.
-
-Interface
-    The antenna diversity module interfaces with FEM, which controls the antennas through a single pin used for antenna selection.
-    The ANT pin of the chip should be connected to the RF interface of FEM.
-    Such simple interface allows even for simple analog switch to act as FEM for antenna diversity.
-    If FEM provides any other pins (for example, for enabling chip, configuration, or other operations), they are not handled by antenna diversity module.
-
-Modes
-    Antenna diversity module supports all radio operations, although different approach is used for reception-related (RX) operations than for transmission-related (TX) operations.
-    This difference stems from the nature of the operations - the best antenna for RX operations can be determined on operation-to-operation basis, for example by measuring the RSSI.
-
-    On the other hand, TX operations usually require knowing before which antenna is better for a given recipient (for example, based on previous operations), which is out of scope of radio driver and should be handled by a higher level module.
-
-    For these reasons, the interface of antenna diversity is split into two parts, responsible for TX and RX operations, respectively.
-    Each of them can be individually disabled or enabled, and the antenna manually selected.
-    Additionally, RX operations support automatic mode, where optimal antenna is selected for each individual operation.
+For more information, read the `antenna diversity documentation`_.
 
 Configuring antenna diversity
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The antenna diversity module is enabled by default for TX operations.
-
+In |NCS|, the antenna diversity module is enabled by default for TX operations.
```

Comment:
Maybe we could change it to something like "The antenna for TX operations needs to be controlled manually by the application". Current sentence does not imply there is no auto mode for TX